### ISSUE TITLE
Japanese Vertical cannot be saved as a language source in dropdown box.

### ISSUE THREAD

**Issue by tenpai-git** [eikek/docspell#2688]
Everything has been working great with Japanese Vertical but I ran into an issue while trying to setup a source for it so I could send to it via the Android version.

Within Docspell, under **Sources**, if I **edit** or make a new source and goto the **Language** dropdown box, I select JpnVert. 

When I click "**submit**" it just goes back to the previous entry, Japanese. It doesn't stay on JpnVert.

I tried multiple different languages and this only happened with JpnVert.

@eikek any ideas on why this might happen? I didn't find anything in my previous commits but I also don't remember needing to do anything special for Sources. 

This is an important part of the usecase since a big part of why this would be useful is taking mobile pictures and sending it to docspell via a language source api on mobile to get vertical text out of books.

**Comment by eikek** [comment]
Thanks for reporting, I'll have a look later. It is likely something related how `JpnVert` is defined (in the list, creating it from a string etc).

**Comment by tenpai-git** [comment]
Thank you for the acknowledgement @eikek - I hope a fix is easy enough to do. It's very hard to use the feature on mobile (which makes it easier to study) without the source being select-able.

**Comment by eikek** [comment]
Hi @tenpai-git , sorry for the late testing. I just started the current master branch and tried to reproduce, but couldn't. When I create a source, chose `JpnVert` and then click submit, it stays to `JpnVert`. Reloading the page etc all is fine. The database also has the correct entry. Could it perhaps be that you had some cache problem? I really can imagine a convincing cause right now…

**Comment by tenpai-git** [comment]
Hi @eikek -

After looking into it, I did find that there must have been some browser-related error, probably from testing multiple different versions or something or another. Re-installing it and putting it on a fresh browser solved this issue. I should have set each version in a different browser profile for proper testing, sorry about that. 

On the bright side, I tested nightly again, and it works perfectly. I think the feature is ready for full release now. :) 

Thank you for looking into this.
